tests/testValues.ts

```diff
@@ -112,3 +112,17 @@ export const test_tattooUint8Array = new Uint8Array([
+
+/** A longer text including very unusual symbols */
+export const test_complicated = `Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam erat, sed diam voluptua. At vero eos et accusam et justo duo dolores et ea rebum. Stet clita kasd gubergren, no sea takimata sanctus est Lorem ipsum dolor sit amet. Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam erat, sed diam voluptua. At vero eos et accusam et justo duo dolores et ea rebum. Stet clita kasd gubergren, no sea takimata sanctus est Lorem ipsum dolor sit amet. Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam erat, sed diam voluptua. At vero eos et accusam et justo duo dolores et ea rebum. Stet clita kasd gubergren, no sea takimata sanctus est Lorem ipsum dolor sit amet.   
```

Comment:
I'm thinking through updating the tests, and the data used - this is a good idea, however it doesn't need the symbols - unicode is tested elsewhere so having a reproducible source text is good though :-)

Reply:
I've got a new chunk of Lorem Ipsum that will be coming in later today - but this is good until then!